pkg/kontainer-engine/types/capabilities.go

```diff
@@ -11,23 +11,3 @@ const (
 func (c *Capabilities) AddCapability(cap int64) {
 	c.Capabilities[cap] = true
 }
-
-func (c *Capabilities) HasGetVersionCapability() bool {
```

Comment:
I realize these functions aren't used anywhere in the `rancher/rancher` repo, but I see some other `rancher` repos import this package. Maybe we should leave them in until kev1 is completely deprecated? They aren't specific to the drivers being removed anyways.